Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:latest
+LABEL authors="lucasliebe"
+
+WORKDIR /app
+
+RUN apt-get update -y && apt-get install default-jre make git wget curl unzip build-essential -y
+RUN git clone --recurse-submodules https://github.com/hpicgs/unCover
```

Comment:
This is rather unusual, is there a reason you clone the repository again instead of just copying the already cloned repository that this Dockerfile is part of onto the image?

Reply:
I wrote the Dockerfile assuming only this one is downloaded, many other repositories I work with these days follow that approach, however we could also copy it if you prefer it

Reply:
Interesting! I think it would be best to publish the image to a hub like Docker Hub, that way people don't need the file at all. Or if someone really wants to build the image themselves then they'd probably clone the whole repo anyways.

I'd personally prefer the copying. But if you have a strong opinion I'm also fine with leaving it as it is.

Reply:
Ok, I will make the required changes.